src/DynamoCore/Core/Threading/UpdateGraphAsyncTask.cs

```diff
@@ -25,7 +25,7 @@ internal override TaskPriority Priority
             get { return TaskPriority.AboveNormal; }
         }
 
-        internal IEnumerable<NodeModel> ModifiedNodes { get; private set; }
+        public IEnumerable<NodeModel> ModifiedNodes { get; protected set; }
```

Comment:
I need this member to be filled up by the derived class `FakeUpdateGraphAsyncTask` so that it won't be `null` when the base method `UpdateGraphAsyncTask.CanMergeWithCore` is called (this data member will never be `null` in the usual `UpdateGraphAsyncTask`).